readme.md

```diff
@@ -20,7 +20,9 @@ Before you begin installing make sure you run `sudo apt update` to get the lates
 sudo apt install mysql-server
 
 # php & required extensions
-sudo apt install php7.2 php7.2-mysql php7.2-xml php7.2-mbstring php7.2-bcmath php7.2-sqlite php7.2-json
+sudo add-apt-repository ppa:ondrej/php
```

Comment:
Why PPA?

Reply:
because we are very specific about the php version 7.3